api/src/wfl/api/handlers.clj

```diff
@@ -146,7 +146,7 @@
                            (jdbc/query tx ["SELECT uuid FROM workload"]))))]
        (util/do-or-nil (postgres/update-workload-for-uuid! tx uuid-vec))
        (let [result (mapv (partial postgres/get-workload-for-uuid tx) uuid-vec)]
-         (if (every? empty? result)
+         (if (and (not-empty result) (every? empty? result))
```

Comment:
How about only calling `update-workload-for-uuid!` and `get-workload-for-uuid` if `uuid-vec` is not empty? That might make this logic a bit clearer 🤔

Reply:
Good idea!